docs/developer/how-to/make-new-ophyd-async-device.md

```diff
@@ -0,0 +1,58 @@
+# Decision Flowchart for Creating a New ophyd_async Device
+
+This document contains decision flowcharts designed to guide developers through the process of creating a new ophyd_async device in the Ophyd library. These flowcharts help in determining the appropriate class inheritance, methods to override, and testing procedures for optimal device functionality and integration into the system.
+
+## High-Level Development Flowchart
+
+This high-level flowchart guides the overall process of creating a new ophyd_async device, from testing PVs to overriding methods and determining state enums.
+
+```{mermaid}
+
+
+  flowchart TD
+    highLevelStart([Start]) --> scoutUsers[Scout current and potential users of the device, tag them in the draft PR]
+    scoutUsers --> testPVs[Test PVs to get the right values]
+    testPVs --> decideStateEnums[Decide on State Enums: extend str, Enum]
+    decideStateEnums --> chooseMethods[Choose which superclass methods to override]
+    chooseMethods --> finalizeDevice[Finalize Device Implementation]
+    finalizeDevice --> requestFeedback[Request feedback from tagged users]
+    requestFeedback --> rebaseOnMain[Rebase on main -> make sure tests pass still]
+    rebaseOnMain --> coordinateMerges[Coordinate merges with other codebase updates]
+    coordinateMerges --> markPRReady[Mark PR as ready]
+
+```
+
+## Interface Selection Flowchart
+
+This flowchart assists in selecting the appropriate interfaces based on the device's capabilities, such as file writing, reading values from process variables (PVs), and mobility within scans.
```

Comment:
Nit: I don't think this adds anything, it just reiterates the section title with a few more words. If it were me I'd get rid of it.

Reply:
deleted